src/create-index.ts

```diff
@@ -69,18 +69,19 @@ export type IndexType = {
     keyItemGroupMap: Map<Midashi, ItemGroup[]>;
     SudachiSynonymsItemGroup: Map<SudachiSynonyms, ItemGroup>;
 };
-let _ret: IndexType | null = null;
-export const createIndex = async (): Promise<IndexType> => {
-    if (_ret) {
-        return Promise.resolve(_ret);
+let _cache = new Map<boolean, IndexType>();
+export const createIndex = async ({ allowLexeme }: { allowLexeme: boolean }): Promise<IndexType> => {
+    if (_cache.has(allowLexeme)) {
+        return Promise.resolve(_cache.get(allowLexeme)!);
     }
     assertInstallationSudachiSynonymsDictionary();
     const keyItemGroupMap: Map<Midashi, ItemGroup[]> = new Map();
     const SudachiSynonymsItemGroup: Map<SudachiSynonyms, ItemGroup> = new Map();
     const SynonymsDictionary = await fetchDictionary();
     SynonymsDictionary.forEach((group) => {
         const groupByVocabularyNumber = group.items.reduce((res, item) => {
-            res[item.vocabularyNumber!] = (res[item.vocabularyNumber!] || []).concat(item);
+            const vocabularyNumber = allowLexeme ? item.vocabularyNumber! : 1;
```

Comment:
I want to avoid hard code `1`.
Please use variable which define `1` as a value.